task_1.py

```diff
@@ -0,0 +1,16 @@
+# зробив 2 прирівняння бо свариться(пропонує перейменувати)
```

Comment:
nit: better to name inputs like:
`argument_input_data`
`power_of_argument_input_data`